league/match.go

```diff
@@ -0,0 +1,458 @@
+package league
+
+import "fmt"
+
+//MatchService handles getting match data
+type MatchService service

```

Comment:
Can we wrap all the types in a `type( )`? There’s an example in the ddragon package.